config/runtime.exs

```diff
@@ -0,0 +1,170 @@
+import Config
+
+# config/runtime.exs is executed for all environments, including
+# during releases. It is executed after compilation and before the
+# system starts, so it is typically used to load production configuration
+# and secrets from environment variables or elsewhere. Do not define
+# any compile-time configuration in here, as it won't be applied.
+# The block below contains prod specific runtime configuration.
+
+# ## Sensitive values should be passed through the environment. An example
+# EnvironmentFile that can be used with systemd is under
+# documents/prod_files/example-environment-file
+
+# ## Using releases
+#
+# If you use `mix release`, you need to explicitly enable the server
+# by passing the PHX_SERVER=true when you start it:
+#
+#     PHX_SERVER=true bin/teiserver start
+#
+# Alternatively, you can use `mix phx.gen.release` to generate a `bin/server`
+# script that automatically sets the env var above.
+if Teiserver.ConfigHelpers.get_env("PHX_SERVER", nil) do
+  config :teiserver, TeiserverWeb.Endpoint, server: true
+end
+
+# Only do some runtime configuration in production since in dev and tests the
+# files are automatically recompiled on the fly and thus, config/{dev,test}.exs
+# are just fine
+if config_env() == :prod do
+  # used for mailing, checking origins, finding tls certs…
+  domain_name = Teiserver.ConfigHelpers.get_env("DOMAIN_NAME", "beyondallreason.info")
+
+  certificates = [
+    keyfile: "/etc/letsencrypt/live/#{domain_name}/privkey.pem",
+    certfile: "/etc/letsencrypt/live/#{domain_name}/cert.pem",
+    cacertfile: "/etc/letsencrypt/live/#{domain_name}/fullchain.pem"
```

Comment:
All those locations need to be configurable, those paths effectively assume certbot usage which is invalid.

Reply:
I amended the commit, the 3 paths are now expected to be in an environment variable.